customerio/api.py

```diff
@@ -0,0 +1,15 @@
+from .client_base import ClientBase
+
+
+class APIClient(ClientBase):
+
+    def __init__(self, key, api_key=None, url=None, retries=3, timeout=10, backoff_factor=0.02):
+        self.url = url or 'https://api.customer.io'
+        ClientBase.__init__(self, retries=retries,
+                            timeout=timeout, backoff_factor=backoff_factor)
+        self.http.headers = {
+            "Authorization": "Bearer {key}".format(key=key)}
+
+    def send_email(self, customer_id, **kwargs):
```

Comment:
We should also have to update readme to reflect this new transactional addition.

Reply:
To keep the rest of our libraries consistent, I think we should get rid of `customer_id` as a positional argument and instead just expect it as `kwargs` as well. That's what our Go, Ruby, and Node libraries do.